src/components/SalaryInfo/SalaryInfo.jsx

```diff
@@ -0,0 +1,33 @@
+import "./SalaryInfo.scss";
+
+const SalaryInfo = ({ avgSalaryNumber, salariesObj }) => {
+  const titleJSX = Object.keys(salariesObj).map((role) => {
+    return (
+      <h6 key={role} className="salary__title">
+        {role}
+      </h6>
+    );
+  });
+
+  const payJSX = Object.values(salariesObj).map((salary) => {
+    return (
+      <h6 key={salary} className="salary__pay">
```

Comment:
I don't think salary will be a good key, as it might be the same for few things. We won't delete or edit that data, so using index instead will be fine

Reply:
Changed salary key to index